test/io/mandoline/impl/cache_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns io.mandoline.impl.cache-test
+  (:use clojure.test
+        clojure.test io.mandoline.backend.mem)
```

Comment:
nit: `clojure.test` twice? Also why :use at all after requiring `io.mandoline.backend.mem :as mem` and `clojure.test :refer :all` already?